Vester/Public/New-VesterConfig.ps1

```diff
@@ -149,6 +153,7 @@ function New-VesterConfig {
     $vCenter    = $DefaultVIServers.Name
     $Datacenter = Get-Datacenter -Name $config.scope.datacenter -Server $vCenter
     $Cluster    = $Datacenter | Get-Cluster -Name $config.scope.cluster
+    $DSCluster  = Get-DatastoreCluster -Name $config.scope.dscluster
```

Comment:
If we pipe in $Datacenter, it would need to exist here as well.  i.e. $DSCluster = $Datacenter | Get-DatastoreCluster -Name $config.scope.dscluster